MediaGalleryUi/Model/Listing/DataProvider.php

```diff
@@ -73,7 +83,6 @@ public function getData(): array
         try {
             /** @var SearchResult $searchResult */
             $searchResult = $this->getSearchResult();
-            $this->selectModifier->apply($searchResult->getSelect(), $this->getSearchCriteria());
 
             return $this->searchResultToOutput($searchResult);
```

Comment:
Unnecessary variable can be eliminated
```suggestion
            return $this->searchResultToOutput($this->getSearchResult());
```